Bio-page.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alexander Beaumier - Personal Website</title>
+    <link rel="stylesheet" href="css/Styles2.css"> 
```

Comment:
consider renaming these style files to be a bit more descriptive of what they are holding - for example maybe 'bio-styles.css' or 'portfolio-styles.css'.